toolchain/sem_ir/generic.h

```diff
@@ -29,6 +31,74 @@ struct Generic : public Printable<Generic> {
   InstBlockId bindings_id;
 };
 
+// An instance of a generic entity, such as an instance of a generic function.
+// For each construct that depends on a compile-time parameter in the generic
+// entity, this contains the corresponding non-generic value. This includes
+// values for the compile-time parameters themselves.
+struct GenericInstance : Printable<GenericInstance> {
+  // Values corresponding to a region of a generic.
```

Comment:
What's a region of a generic? I don't see the term used in the design.

Reply:
Sorry, I'd incompletely removed this. Will be introduced in a future PR, gone for now :)